docs/querying.md

```diff
@@ -93,6 +93,22 @@ Post.findAll({
 });
 // SELECT * FROM post WHERE authorId = 12 AND status = 'active';
 
+Post.findAll({
+    where: {
```

Comment:
Other examples are using 2 space indentation, Please use that in these new examples as well